CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 2.2.2-preview (Under active development)
 
+### App Center Analytics
+
+#### WPF/WinForms
+
+* **[Fix]** Fix reporting the correct device models when a placeholder was used in some cases.
```

Comment:
I think it is supposed to be like this:
```suggestion
* **[Fix]** Fix reporting the incorrect device models when a placeholder was used in some cases.
```
Since we fix sending incorrect device model, not fix sending what is actually correct?